poutyne/framework/metrics/epoch_metrics/__init__.py

```diff
@@ -0,0 +1,14 @@
+# pylint: disable=wildcard-import
+from .base import *
+from .fscores import *
+
+all_epochs_metrics_dict = dict(f1=F1)
+
+
+def get_epoch_metric(epoch_metric):
```

Comment:
In the __init__, why here ? 

> If we apply this logic, we should do the same for batch metric

Reply:
Because there other is some kind of cyclic import. But in the future, maybe we should use decorator instead of a dictionnary.

Reply:
Clearly a decorator is more appropriated.

Reply:
Yes, but we'll do that later.